multiqc/modules/supernova/supernova.py

```diff
@@ -0,0 +1,440 @@
+        self.headers['Asm size'] = {
+                'description': 'assembly size (in megabases) ;only scaffolds >= 10 kb',
+                'modify': lambda x: x / 1000000.0,
+                'suffix': 'Mb',
+                'scale': 'YlGn'
+        }
+        self.headers['# Long scaffs'] = {
+                'description': 'number of scaffolds >= 10 kb',
+                'scale': 'YlGn',
+                'format': '{:,.0f}',
+        }
+        self.headers['Scaff N50'] = {
+                'description': 'N50 scaffold size (in kilobases)',
+                'modify': lambda x: x / 1000.0,
+                'suffix': 'Kb',
+                'scale': 'RdYlGn'
+        }
+        self.headers['Phase N50'] = {
+                'description': 'N50 phase block size (in kilobases)',
+                'modify': lambda x: x / 1000.0,
+                'suffix': 'Kb',
+                'scale': 'RdYlGn',
+                'hidden': True
+        }
+        self.headers['Contig N50'] = {
+                'description': 'N50 contig size (in kilobases)',
+                'modify': lambda x: x / 1000.0,
+                'suffix': 'Kb',
+                'scale': 'RdYlGn',
+                'hidden': True
+        }
+        self.headers['Edge N50'] = {
+                'description': 'N50 edge size (in kilobases)',
+                'modify': lambda x: x / 1000.0,
+                'suffix': 'Kb',
+                'scale': 'RdYlGn',
+                'hidden': True
+        }
+        self.headers['Mol size'] = {
+                'description': 'weighted mean molecule size (in kilobases); ideal 50-100',
+                'modify': lambda x: x / 1000.0,
+                'suffix': 'Kb',
+                'scale': 'BuGn'
+        }
+        self.headers['Read len'] = {
+                'description': 'mean read length (in bases) after trimming; ideal 140',
+                'suffix': 'b',
+                'scale': 'PuBu',
+                'format': '{:,.0f}',
+                'hidden': True
+        }
+        self.headers['# Reads'] = {
+                'description': 'number of reads (in millions); ideal 800M-1200M for human',
+                'modify': lambda x: x / 1000000.0,
+                'suffix': 'M',
+                'scale': 'PuBu',
+        }
+        self.headers['Coverage'] = {
+                'description': 'effective read coverage; ideal ~42 for nominal 56x cov',
+                'suffix': 'x',
+                'scale': 'PuBu'
+        }
+        self.headers['% Dup'] = {
+                'description': 'fraction of reads that are duplicates',
+                'suffix': '%',
+                'scale': 'OrRd',
+        }
+        self.headers['% R2 Q30'] = {
+                'description': 'fraction of Q30 bases in read 2; ideal 75-85',
+                'suffix': '%',
+                'scale': 'OrRd',
+        }
+        self.headers['Insert size'] = {
+                'description': 'median insert size (in bases); ideal 0.35-0.40',
```

Comment:
Do you mean `0.35-0.40` here?

Reply:
Aha! This text was mostly cut 'n pasted from the `report.txt` files of supernova. But it would make more sense to change this to `350-400` now